.github/workflows/yices2.yml

```diff
@@ -0,0 +1,34 @@
+name: 'yices2'
```

Comment:
Some fixes were pushed a few minutes ago. Please compare with https://github.com/hdl/containers/blob/main/.github/workflows/z3.yml. Probably, just copy it again and `sed 's/z3/yices2/'`.

Reply:
Done.